frontend/src/components/ShootVersion.vue

```diff
@@ -72,6 +72,7 @@ limitations under the License.
         ></shoot-version-update>
         <template v-if="!selectedVersionInvalid && selectedVersionType === 'minor'">
           You should always test your scenario and back up all your data before attempting an upgrade. Don’t forget to include the workload inside your cluster!<br /><br />
+          You should consider the <a href="https://github.com/kubernetes/kubernetes/releases" target="_blank">Kubernetes release notes</a> before upgrading your cluster.<br /><br />
```

Comment:
please add the open-in-new-icon as we do it e.g. here https://github.com/gardener/dashboard/blob/master/frontend/src/dialogs/SecretDialogAwsHelp.vue#L34